odoo/addons/base/models/res_users.py

```diff
@@ -1797,7 +1797,7 @@ def linearize(app, gs, category_name):
             # children of Accountant, but the two of them make a full accountant
             # so it makes no sense to have checkboxes.
             if app.xml_id == 'base.module_category_accounting_accounting':
-                return (app, 'selection', gs.sorted(key=order.get), category_name)
+                return (app, 'selection', gs.sorted(key=lambda g: 50 if g.get_external_id()[g.id] == 'account.group_account_readonly' else order.get(g)), category_name)
```

Comment:
I don't think this is needed.
The function groups by application category, so it should not look at xml ids of groups.

Reply:
I wanted to get one group to appear last in the selection dropdown, even though it has fewer implied groups.
I mentioned to @chklop that it wasn't really possible...(unless we change the implied group)
but I was just trying this ugly hack.
(I also see now that I could do `gs.get_external_id()` instead)